src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Circle {
```

Comment:
The class declaration should not be on the same line as the package declaration. Please move the class declaration to a new line.